### PR TITLE
[ja] Fix translation

### PR DESCRIPTION
**Pull request by shun-fix9**

### CODE REVIEW

File: ja/SUMMARY.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
     * [IETF](proc-ietf.md)
     * [HTTP/2 からの経験](proc-h2.md)
     * [現在の状況](proc-status.md)
-* [プロコトルの機能](the-protocol.md)
+* [プロトコルの機能](the-protocol.md)
```
Comment (inductor) [inline ja/SUMMARY.md]: What’s the difference?

File: ja/feature-http.md [hunk 2]
```diff
@@ -4,5 +4,5 @@ HTTP レイヤーは HTTP に準拠したトランスポートを行います。
 これは HTTP/2 における HPACK に似ています。
 
 HPACK アルゴリズムはストリームが順番に届くことを前提としているため、修正せずに HTTP/3 で再利用することは不可能です。
-これは QUIC の提供するストリームにおいては伝送がばらばらに行われるためです。
+これは QUIC の提供するストリームにおいては転送がばらばらに行われるためです。
```
Comment (inductor) [inline ja/feature-http.md]: What’s the difference?

File: ja/feature-nonhttp.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 ## HTTP 以外のプロトコルを QUIC 上で動かす
 
-HTTP 以外のプロトコルを QUIC を使って通信するための対応は、QUIC のバージョン1が実際にリリースされた後に行われるよう延期されました。
\ No newline at end of file
+HTTP 以外のプロトコルを QUIC を使って通信するための対応は、QUIC のバージョン1が実際にリリースされた後に行われるよう延期されました。
```
Comment (inductor) [inline ja/feature-nonhttp.md]: Why changed?

File: ja/proc-status.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Apache や nginx が QUIC をサポートしたという公式な発表はあり
 
 ## クライアント
 
-IETF バージョンの QUIC や HTTP/3 をサポートしたブラウザをリリースした大規模ベンダはまだありません。
+IETF バージョンの QUIC や HTTP/3 をサポートしたブラウザをリリースした大規模ベンダーはまだありません。
```
Comment (inductor) [inline ja/proc-status.md]: What’s the difference?

File: ja/quic-streams.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ QUIC におけるストリームは軽量で順序付けられたバイトスト
 
 QUIC には2種類の基本的なストリームが存在します:
 
- - イニシエータからピアへデータを1方向に伝送する単方向ストリーム
+ - イニシエータからピアへデータを1方向に転送する単方向ストリーム
```
Comment (inductor) [inline ja/quic-streams.md]: What’s the difference?

File: ja/feature-http.md (hunk 2 above)
Comment (shun-fix9) [reply ja/feature-http.md]: fix: 伝送 -> 転送

File: ja/quic-streams.md (hunk 5 above)
Comment (shun-fix9) [reply ja/quic-streams.md]: fix: 伝送 -> 転送

File: ja/proc-status.md (hunk 4 above)
Comment (shun-fix9) [reply ja/proc-status.md]: fix: ベンダ -> ベンダー

File: ja/the-protocol.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
-# プロコトルの機能
+# プロトコルの機能
 
-高レベルからの QUIC プロコトル
+高レベルからの QUIC プロトコル
```
Comment (inductor) [inline ja/the-protocol.md]: What’s the difference?

File: ja/quic-streams.md [hunk 7]
```diff
@@ -56,4 +56,4 @@ HTTP/2 のような他の多重化されたプロトコルでの経験から言
 QUIC 自身は優先順位付けの情報を交換するフレームを持ちません。そのかわり、QUIC を利用するアプリケーションからの優先順位情報を信頼します。
 QUIC を利用するプロトコルはそのアプリケーションのセマンティクスに合った優先順位付けのスキームを定義することが出来ます。
 
-HTTP/3 に QUIC を利用する場合、HTTP レイヤーにおいては優先順位付けは不要です。
\ No newline at end of file
+HTTP/3 に QUIC を利用する場合、HTTP レイヤーにおいては優先順位付けは不要です。
```
Comment (inductor) [inline ja/quic-streams.md]: Why changed?

File: ja/feature-nonhttp.md (hunk 3 above)
Comment (shun-fix9) [reply ja/feature-nonhttp.md]: There is no `newline` at the end of file in this file.
(All other `ja/*.md` have `newline` at the end of file.)

File: ja/SUMMARY.md (hunk 1 above)
Comment (shun-fix9) [reply ja/SUMMARY.md]: fix: `プロ(コト -> トコ)ル`

File: ja/the-protocol.md (hunk 6 above)
Comment (shun-fix9) [reply ja/the-protocol.md]: fix: プロ(コト -> トコ)ル

**Pull request merged**